pkg/kubelet/kubelet.go

```diff
@@ -1541,7 +1541,10 @@ func (kl *Kubelet) syncPod(o syncPodOptions) error {
 	existingStatus, ok := kl.statusManager.GetPodStatus(pod.UID)
 	if !ok || existingStatus.Phase == v1.PodPending && apiPodStatus.Phase == v1.PodRunning &&
 		!firstSeenTime.IsZero() {
-		metrics.PodStartDuration.Observe(metrics.SinceInSeconds(firstSeenTime))
+		if !pod.CreationTimestamp.IsZero() {
```

Comment:
Can you add a comment about when we expect this to be zero?  Or is this just from an abundance of caution?

Reply:
will make a comment, it's just for caution. Theoretically api server shall fill in creation timestamp